content/en/news/releases/1.14.x/announcing-1.14/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Announcing Istio 1.14
+linktitle: 1.14
+subtitle: Major Update
+description: Istio 1.14 release announcement.
+publishdate: 2022-05-11
+release: 1.14.0
+skip_list: true
+aliases:
+- /news/announcing-1.14
+- /news/announcing-1.14.0
+---
+
+We are pleased to announce the release of Istio 1.14!
+
+{{< relnote >}}
+
+This is the second Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.14.0 published.
+Special thanks are due to the release managers Lei Tang (Google) and Greg Hanson (solo.io),
+and to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
+
+{{< tip >}}
+Istio 1.14.0 is officially supported on Kubernetes versions `1.20` to `1.23`.
+{{< /tip >}}
+
+Here are some of the highlights of the release:
+
+## Add a new approach for CA integration through the Envoy SDS API
+
+This new feature adds a new approach for CA integration through the Envoy SDS API.
+For more information, check out the [documentation](/docs/ops/integrations/spire/).
+
+## Add the auto-sni support
```

Comment:
@kfaseela Do you have a user guide (e.g., a link on the istio.io/docs) for this feature?

Reply:
It's part of https://github.com/istio/istio.io/pull/11057 which hasn't merged yet.

Reply:
@ericvn Correct me if I am wrong. I think the #11057 is for the new feature of CA integration through the Envoy SDS API.
My first comment is about the user guide for the auto-sni feature.

Reply:
You are correct.

Reply:
Hello @lei-tang ,  this was a small enhancement to the existing DestinationRule logic, so I have not really prepared any documentation changes for the same. May be I can enhance the sni section at https://istio.io/latest/docs/reference/config/networking/destination-rule/ with additional description of the new feature flag, let me know.

Reply:
Your plan sounds good to me. After you complete the documentation, please let me know which link you would like to include in this PR. Thanks!

Reply:
Ok, the documentation change for auto_sni is here:  https://github.com/istio/api/pull/2338
and the page that would get updated will be : https://istio.io/latest/docs/reference/config/networking/destination-rule/

Reply:
Thanks! I have added the link https://istio.io/latest/docs/reference/config/networking/destination-rule/ to the PR.

Reply:
@ericvn May be you could get me a review on the above docs PR?

Reply:
https://github.com/istio/api/pull/2338 has been merged.